active_directory_ldap/komand_active_directory_ldap/actions/modify_groups/action.py

```diff
@@ -18,19 +19,35 @@ def __init__(self):
     def run(self, params={}):
         conn = self.connection.conn
         dn = params.get('distinguished_name')
+        group_dn = params.get('group_dn')
+        add_remove = params.get('add_remove')
+
+        # Normalize dn
         dn = ADUtils.dn_normalize(dn)
         temp_list = ADUtils.dn_escape_and_split(dn)
         dn = ','.join(temp_list)
-        group_dn = params.get('group_dn')
-        add_remove = params.get('add_remove')
+        # Normalize group dn
+        group_dn = ADUtils.dn_normalize(group_dn)
+        temp_list = ADUtils.dn_escape_and_split(group_dn)
+        group_dn = ','.join(temp_list)
 
         if add_remove == 'add':
-            group = extend.ad_add_members_to_groups(conn, dn, group_dn)
+            try:
+                group = extend.ad_add_members_to_groups(conn, dn, group_dn)
+            except LDAPInvalidDnError as e:
+                raise PluginException(cause='Either the user or group distinguished name was not found.',
+                                      assistance='Please check that the distinguished names are correct',
```

Comment:
The cause and assistance here are re-used, it would be good to make these strings constants